### PR TITLE
Script change

### PR DESCRIPTION
**Pull request by s-varun**

### CODE REVIEW

File: db/install.sh [hunk 1]
```diff
@@ -3,11 +3,14 @@
...
 
 sudo -u $POSTGRES_ADMIN createdb -O $DBUSER $DBNAME
 hostName=$(sudo netstat -plunt |grep postgres)
 echo $hostName | cut -d " " -f 4
 
-psql -d $DBNAME -c $"CREATE TABLE IF NOT EXISTS $TABLENAME ( url TEXT PRIMARY KEY NOT NULL, metadata TEXT, created_date datetime default NULL );"
+psql -d $DBNAME -c $"CREATE TABLE IF NOT EXISTS $TABLENAME ( \
+                        url TEXT PRIMARY KEY NOT NULL, \
+                        metadata TEXT, \
+                        created_date timestamp);"
```
Comment (AliceYuan) [inline db/install.sh]: this should be
timestamp without time zone

**Comment by AliceYuan** [comment]
r+ with change

File: db/install.sh [hunk 2]
```diff
@@ -3,11 +3,14 @@
 
 POSTGRES_ADMIN=postgres
 DBUSER=postgres
-DBNAME=postgres_magpie
+DBNAME=postgres_db
 TABLENAME=magpie_cache
 
 sudo -u $POSTGRES_ADMIN createdb -O $DBUSER $DBNAME
```
Comment (samiyaakhtar) [inline db/install.sh]: `createdb -O $DBUSER $DBNAME`
This is what works on my machine

File: db/install.sh [hunk 3]
```diff
@@ -3,11 +3,14 @@
...
 hostName=$(sudo netstat -plunt |grep postgres)
```
Comment (samiyaakhtar) [inline db/install.sh]: I get an error from this line: "netstat: lunt: unknown or uninstrumented protocol"

**Pull request closed without merging**